packages/twenty-front/src/modules/activities/files/utils/uploadFile.ts

```diff
@@ -0,0 +1,39 @@
+import { getFileType } from '@/activities/files/utils/getFileType';
+import { ActivityTargetableObject } from '@/activities/types/ActivityTargetableEntity';
+import { getActivityTargetObjectFieldIdName } from '@/activities/utils/getTargetObjectFilterFieldName';
+import { FileFolder } from '~/generated/graphql';
+
+export const uploadAttachFile = async (
```

Comment:
Could this be turned into an `useUploadAttachmentFile` hook, that would be moved to `/src/modules/activities/files/hook`?

This way these lines can be placed in the hook:
```
  const currentWorkspaceMember = useRecoilValue(currentWorkspaceMemberState);
  const [uploadFile] = useUploadFileMutation();

  const { createOneRecord: createOneAttachment } =
    useCreateOneRecord<Attachment>({
      objectNameSingular: CoreObjectNameSingular.Attachment,
    });
```

The `useUploadAttachmentFile` hook would return an async `uploadAttachmentFile` function that can be used in the components.